File: app/src/main/java/ir/beigirad/zeroapplication/ZeroApplication.java
package ir.beigirad.zeroapplication;

import android.app.Application;
import android.content.Context;

import com.facebook.stetho.Stetho;

/**
 * Created by farhad-mbp on 7/31/17.
 */

public class ZeroApplication extends Application {
    private static Context mContext;

    @Override
    public void onCreate() {
        super.onCreate();
        mContext=getApplicationContext();


        Stetho.initializeWithDefaults(this);
    }

    public static Context getAppContext(){
        return mContext;
    }
}

File: app/src/main/java/ir/beigirad/zeroapplication/ZeroApplication.kt
package ir.beigirad.zeroapplication

import android.app.Application
import android.content.Context
import timber.log.Timber

/**
 * Created by farhad-mbp on 11/4/17.
 */

open class ZeroApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        mContext = applicationContext

        Timber.plant(Timber.DebugTree())
    }

    companion object {
        private var mContext: Context? = null

        fun getAppContext(): Context {
            return mContext!!
        }

    }
}
